CakePHP/Tests/Formatting/UseInAlphabeticalOrderUnitTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace CakePHP\Tests\Formatting;
+
+use PHP_CodeSniffer\Tests\Standards\AbstractSniffUnitTest;
+
+class UseInAlphabeticalOrderUnitTest extends AbstractSniffUnitTest
+{
+    /**
+     * Returns the lines where errors should occur.
+     *
+     * The key of the array should represent the line number and the value
+     * should represent the number of errors that should occur on that line.
+     *
+     * @param string $testFile The name of the file being tested.
+     *
+     * @return array<int, int>
+     */
+    public function getErrorList($testFile='')
+    {
+        switch ($testFile) {
+            case 'UseInAlphabeticalOrderUnitTest.1.inc':
+                return [
+                    3 => 1,
+                    4 => 1,
+                    8 => 1,
+                    9 => 1,
+                ];
+
+            default:
+                return [];;
```

Comment:
@josegonzalez Yes there is `Generic.Formatting.DisallowMultipleStatements.SameLine` but I didn't checked the CS yet.
It has been updated and should now be part of the travis build